internal_ws/yktrace/Cargo.toml

```diff
@@ -23,6 +22,9 @@ version = "0.23.0"
 default-features = false
 features = ["read_core", "elf"]
 
+[target.'cfg(tracermode = "hw")'.dependencies]
```

Comment:
Nice! Didn't know you could do that!

Reply:
I just read the cargo docs and it turns out you can't actually. I haven't been testing with hardware tracing enabled. I could make it conditional on a target feature though.

Reply:
As long as it can integrate with the current build system without any extra work on the user's behalf. This could be tricky.

Reply:
Fixed it.